web/impact/impact/v1/helpers/judging_round_helper.py

```diff
@@ -48,3 +49,7 @@ def full_name(self):
         # short_name is "2017-04 BOS CH IL Round 1".  See ticket
         # AC-5694 for proposed cleanup.
         return self.subject.short_name()
+
+    @property
+    def has_criteria(self):
+        return JudgingRound.objects.get(pk=self.subject.id).criterion_set.count() > 0
```

Comment:
is the call to JudgingRound.objects.get necessary? is self.subject itself a judging round instance?

Reply:
👍  Good catch!

Reply:
@kaymation once this addressed we can move to merge